authz/invalid_http_rules.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: networking.gke.io/v1
+kind: GCPAuthzPolicy
+metadata:
+  name: "test"
+spec:
+  action: ALLOW
+  targetRefs:
+  - kind: Deployment
+    name: test-deploy
+    group: ""
+    
+
```

Comment:
Nit: Cleanup trailing new lines in this and other files

Reply:
done